modules/manage/pages/manage-nodes/modify-services-and-rebalance.adoc

```diff
@@ -0,0 +1,154 @@
+= Modify Services and Rebalance
+:description: pass:q[Add or remove non-Data Services on existing nodes in a cluster and rebalance the cluster without adding or removing nodes.]
+:page-toclevels: 3
+
+[abstract]
+{description}
+
+Dynamically add or remove the following Multi-Dimensional Scaling (MDS) services on existing nodes in a cluster from the Couchbase xref:manage:manage-nodes/modify-services-on-nodes-and-rebalance.adoc#modify-mds-services-from-ui[UI], xref:rest-api:rest-set-up-services-existing-nodes.adoc[REST API], or xref:manage:manage-nodes/modify-services-on-nodes-and-rebalance.adoc#modify-mds-services-using-cli[CLI]:
+
+* `index` (Index Service)
+* `n1ql` (Query Service)
+* `fts` (Search Service)
+* `cbas` (Analytics Service)
+* `eventing` (Eventing Service)
+* `backup` (Backup Service)
+
+Then a rebalance operation is automatically triggered to distribute the service workload across the nodes and complete the modification.
+
+You do not have to add or remove a node to add or remove a non-Data Service on a node in a cluster.
+
+NOTE: You cannot add or remove kv (Data Service) using this method.
+Adding or removing of the Data Service on an existing node is supported only through adding or removing nodes.
+For information about adding or removing a Data Service on an existing node, see xref:manage:manage-nodes/manage-data-service-and-rebalance.adoc[Manage a Data Service and Rebalance].
```

Comment:
For information about adding or removing a Data Service on an existing node, see xref:manage:manage-nodes/manage-data-service-and-rebalance.adoc[Adding or Removing the Data Service on existing nodes].